pvm/ts/src/engine/texasHoldem-multiplayer.test.ts

```
@@ -36,7 +36,7 @@ describe("Texas Holdem - Multiplayer", () => {
             expect(game.exists(PLAYER_4)).toBeDefined();
         });
 
-        it("should have correct legal actions after posting the blinds", () => {
+        it.only("should have correct legal actions after posting the blinds", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)